frontend/apps/app/lib/mastra/agents/databaseSchemaAgent.ts

```diff
@@ -0,0 +1,26 @@
+import { openai } from '@ai-sdk/openai'
```

Comment:
In mastra, the documentation says to put the agent under `src/mastra`.

https://mastra.ai/ja/docs/getting-started/installation#%E3%82%A8%E3%83%BC%E3%82%B8%E3%82%A7%E3%83%B3%E3%83%88%E3%81%AE%E7%99%BB%E9%8C%B2